release.sh

```diff
@@ -7,6 +7,15 @@ if [ -z "$1" ]; then
   exit 1
 fi
 
+tools=("helm-docs" "gh" "yq" "git-cliff" "git" "curl" "awk" "sed")
```

Comment:
curl/awk/sed not mentioned in README.md but realistically these should already be installed